### PR TITLE
quicksight dashboard connected to rds metadata database for platform monitoring

### PR DESCRIPTION
**Pull request by dlpzx**
### Feature or Bugfix
- Feature

### Detail
- Automated security groups deployment to enable connection between RDS metadata database and Quicksight
- Updated documentation to add monitoring features
- Updated documentation with steps and information needed for Quicksight RDS datasets

### Relates
#34 
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by dlpzx** [comment]
- WIP: as next step we can include an analysis template and make some of the manual steps automatic

File: backend/dataall/aws/handlers/secrets_manager.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+def ns2d(**kwargs):
+    return kwargs
+
+
+class SecretsManager:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def client(AwsAccountId, region):
```
Comment (LEUNGUU) [inline backend/dataall/aws/handlers/secrets_manager.py:19]: I like this function!

File: backend/dataall/api/Objects/Tenant/resolvers.py [hunk 2]
```diff
@@ -32,3 +38,95 @@ def list_tenant_groups(context, source, filter=None):
             data=filter,
             check_perm=True,
         )
+
+
+def update_ssm_parameter(context, source, name: str = None, value: str = None):
+    current_account = SessionHelper.get_account()
+    region = os.getenv('AWS_REGION', 'eu-west-1')
+    print(value)
```
Comment (LEUNGUU) [inline backend/dataall/api/Objects/Tenant/resolvers.py:46]: Is it for debugging?

File: backend/dataall/aws/handlers/parameter_store.py [hunk 3]
```diff
@@ -23,11 +23,25 @@ def client(AwsAccountId, region):
...
+        if not parameter_name:
+            raise Exception('Parameter name is None')
+        if not parameter_value:
+            raise Exception('Parameter value is None')
+        try:
+            response = ParameterStoreManager.client(
+                AwsAccountId, region
+            ).put_parameter(Name=parameter_name, Value=parameter_value, Overwrite=True)['Version']
+        except ClientError as e:
+            raise Exception(e)
```
Comment (LEUNGUU) [inline backend/dataall/aws/handlers/parameter_store.py:46]: Perhaps we can use try...except....else... to include the 'return' clause in the else block. Just a suggestion.

**Review by LEUNGUU — APPROVED** [review]
Just a few little things. LGTM

File: backend/dataall/api/Objects/Tenant/resolvers.py (hunk 2 above)
Comment (dlpzx) [reply backend/dataall/api/Objects/Tenant/resolvers.py:46]: It stores the dashboard Id and the VPC connection Id. The other alternative was to use the RDS metadata database, but as they are single key-value parameters I opted to store them in the Parameter store

**Pull request merged**